actions/setup-crib-environment/scripts/parse_profiles.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+profiles=$PROFILES
+
+if [ -n "$profiles" ]; then
+  # Convert comma-separated string to array
+  IFS=',' read -r -a profiles_array <<< "$profiles"
+
+  # Initialize result string
+  result=""
+
+  # Loop through each profile in the array
+  for profile in "${profiles_array[@]}"; do
+    result+="-p $profile "
+  done
+
+  # Remove trailing space
+  result=$(echo $result | sed 's/ *$//')
+
+  # Output the result
+  echo "Profiles args: $result"
+  echo "::set-output name=profile_args::$result"
```

Comment:
I think `::set-output` is deprecated?

now it's `"echo profile_args=$result" >> GITHUB_OUTPUT`

Reply:
Beat me

Reply:
fixed